pkg/model/helper.go

```diff
@@ -15,6 +15,7 @@ package model
 
 import (
 	"fmt"
+	"github.com/aws/aws-controllers-k8s/pkg/names"
```

Comment:
nit: this import should be after `aws-sdk-go` import, seprated by an empty line

Reply:
yes, feel free to fix in a future cleanup patch. Merging this PR as-is for now.